src/psyclone/psyir/backend/fortran.py

```diff
@@ -305,28 +305,16 @@ class FortranWriter(LanguageWriter):
     currently PSyIR algorithm code which has its own gen method for
     generating Fortran).
 
-    :param bool skip_nodes: If skip_nodes is False then an exception \
-        is raised if a visitor method for a PSyIR node has not been \
-        implemented, otherwise the visitor silently continues. This is an \
-        optional argument which defaults to False.
-    :param str indent_string: Specifies what to use for indentation. This \
-        is an optional argument that defaults to two spaces.
-    :param int initial_indent_depth: Specifies how much indentation to \
-        start with. This is an optional argument that defaults to 0.
-    :param bool check_global_constraints: whether or not to validate all \
-        global constraints when walking the tree. Defaults to True.
+    :param kwargs: additional keyword arguments provided to the super class.
+    :type kwargs: unwrapped dict.
 
     '''
     _COMMENT_PREFIX = "! "
 
-    def __init__(self, skip_nodes=False, indent_string="  ",
-                 initial_indent_depth=0, check_global_constraints=True):
+    def __init__(self, **kwargs):
         # Construct the base class using () as array parenthesis, and
```

Comment:
This comment would be useful at L69 in c.py (updated appropriately of course).

Reply:
Done